src/main/java/berlin/yuna/nano/services/metric/logic/MetricService.java

```diff
@@ -78,49 +80,49 @@ public void onEvent(final Event event) {
     protected void addMetricsEndpoint(Event event) {
         event
             .ifPresent(EVENT_HTTP_REQUEST, HttpObject.class, request ->
-            {
-                request.pathMatch(Config.getMetricsBaseUrl() + Config.getPrometheusBaseUrl());
-                request.isMethodGet();
-                    final String response = metrics.prometheus();
-                    request.response()
-                        .statusCode(200)
-                        .body(metrics.prometheus())
-                        .headerMap(Map.of(HttpHeaders.CONTENT_TYPE, ContentType.TEXT_PLAIN)).send(event);
-
-            })
+                Optional.ofNullable(prometheusPath)
+                    .filter( request::pathMatch)
+                    .filter(path -> request.isMethodGet())
```

Comment:
can be replaced with lambda

Reply:
`filter(HttpObject::isMethodGet)`
Its complaining about "Non-static method cannot be referenced from a static context", else have to change `isMethodGet` static